app/src/main/java/com/kms/cura/UpdateSpinner.java

```diff
@@ -0,0 +1,8 @@
+package com.kms.cura;
+
+/**
+ * Created by toanbnguyen on 6/7/2016.
+ */
+public interface UpdateSpinner {
+    void iCallBackUpdate();
```

Comment:
Not in Java though. Method name should start with a verb following with the object of that verb. Please refer to Oracle naming convention.